inst/extdata/gcam-usa/EIA_AEO_fuels.csv

```diff
@@ -1,3 +1,6 @@
+# File: EIA_AEO_fuels.csv
+# Title: EIA AEO Fuel Mappings
```

Comment:
It will be useful to be more descriptive here and be explicit that we're mapping EIA fule names to GCAM names.